src/katsdpmodels/rfi_mask.py

```diff
@@ -124,10 +136,15 @@ def from_hdf5(cls: Type[_R], hdf5: h5py.File) -> _R:
             ('max_baseline', 'f8')
         ])
         data = models.require_columns(data, expected_dtype)
-        ranges = astropy.table.QTable(data[...], copy=False)
-        ranges['min_frequency'] <<= u.Hz
-        ranges['max_frequency'] <<= u.Hz
-        ranges['max_baseline'] <<= u.m
+        ranges = astropy.table.Table(data[...], copy=False)
```

Comment:
Out of curiosity, why not leave this as a QTable since you are adding units anyway ?

Reply:
I went round in circles on this code a lot because neither of the APIs do quite what I wanted them to and I was trying different compromises. I think my conclusion was that I wanted the constructor to be able to take any Table (to be more generic), not just a QTable.

Reply:
Fair enough - was just curious :)